meta/main.yml

```
@@ -3,18 +3,26 @@ galaxy_info:
   author: Shane Frasier
   description: Ansible role for installing Nessus.
   company: CISA Cyber Assessments
+  galaxy_tags:
+    - nessus
   license: CC0
-  min_ansible_version: 2.0
+  # With the release of version 2.10, Ansible finally correctly
+  # identifies Kali Linux as being the Kali distribution of the Debian
+  # OS family.  This simplifies a lot of things for roles that support
+  # Kali Linux, so it makes sense to force the installation of Ansible
+  # 2.10 or newer.
+  min_ansible_version: 2.10
+  namespace: cisagov
   platforms:
     - name: Debian
       versions:
         - stretch
         - buster
+        - bullseye
         # Kali linux isn't an option here, but it is based on
         # Debian Testing:
         # https://www.kali.org/docs/policy/kali-linux-relationship-with-debian
-        - bullseye
-  galaxy_tags:
-    - nessus
+        - bookworm
+  role_name: nessus
```

Since these platforms are in the molecule configuration now.

```suggestion
    - name: Ubuntu
      versions:
        - bionic
        - focal
  role_name: nessus
```